src/host/settings.hpp

```diff
@@ -236,5 +238,6 @@ class Settings
     bool _fInterceptCopyPaste;
 
     bool _TerminalScrolling;
+    WCHAR _answerbackMessage[32];

```

Comment:
I believe you need to initialize the array, as it's only set/initialized when the registry value exists.

Is 32 chars sufficient for most use cases? You don't need to be conservative about memory here.

Reply:
Fixed the initialization, thanks.

As for the size, the original hardware terminals had a 30 char limit, so I just added 1 for the null terminator and rounded up to 32. But if you want to pick another number, I'm happy to change it.

Reply:
No, I'm absolutely fine with 32. 🙂